config.py

```diff
@@ -41,7 +41,6 @@ class Config:
     DECRYPTION_KEY_PASSPHRASE = os.getenv('DECRYPTION_KEY_PATH', 'test')
 
     ACTION_RULE_DELAY_MINUTES = float(os.getenv('ACTION_RULE_DELAY_MINUTES', 0.3))
-    PRINT_FILE_TIME_LIMIT_MINUTES = float(os.getenv('PRINT_FILE_TIME_LIMIT_MINUTES', 10))
```

Comment:
This can probably be removed from the README and `kubernetes.env` too

Reply:
Looks like the `ACTION_RULE_DELAY_MINUTES` is unused too, I'll scrap it while I'm here!